Dockerfile

```diff
@@ -10,10 +10,15 @@ RUN apt-get update && \
     apt-get install python3-catkin-tools -y
 
 # Install ROS dependencies
-RUN apt-get install -y ros-noetic-rviz \
+RUN apt-get install -y --fix-missing \
+        ros-noetic-rviz \
         ros-noetic-xacro \
         ros-noetic-robot-state-publisher \
-        ros-noetic-joint-state-publisher-gui
+        ros-noetic-joint-state-publisher-gui \
+        ros-noetic-ros-control \
+        ros-noetic-ros-controllers \
+        ros-noetic-gazebo-ros-pkgs \
+        ros-noetic-rqt*
 
```

Comment:
These dependencies take too much time to build. Also I got an error when running the docker file. may I suggest breaking them up in two instructions and remove the `--fix-missing` option. 

```suggestion
# Install catkin tools
RUN apt-get update && \
    apt-get install python3-catkin-tools -y

# Install ROS dependencies for visualization
RUN apt-get install -y \
        ros-noetic-rviz \
        ros-noetic-xacro \
        ros-noetic-robot-state-publisher \
        ros-noetic-joint-state-publisher-gui 
```

Reply:
I have tested with this suggestion, and I found that it is better to try with another docker base image, specifically the osrf image. I will do a commit outside this suggestion because of this one does not support to request changes.